error/error.go

```diff
@@ -67,15 +67,15 @@ func ParseLevel(level string) (Level, error) {
 	case "SHOULDNOT":
 		fallthrough
 	case "RECOMMENDED":
-		fallthrough
-	case "NOTRECOMMENDED":
 		return Should, nil
+	case "NOTRECOMMENDED":
+		fallthrough
 	case "MUST":
 		fallthrough
 	case "MUSTNOT":
 		fallthrough
 	case "SHALL":
-		fallthrough
+		return Must, nil
```

Comment:
I don't think we need to make this change.  As it stands in master, `SHALL` falls through to `SHALLNOT`, which falls through to `REQUIRED`, which returns `Must`.  That's [the appropriate level for `SHALL`][1].

Hard-coding `Must` here is fine, but if we want to go with that approach I'd recommend dropping `fallthrough` from this whole `switch` for consistency.

[1]: https://tools.ietf.org/html/rfc2119#section-1

Reply:
Sorry, I made a mistake.